ocaml/typing/untypeast.ml

```diff
@@ -348,7 +348,8 @@ let pattern : type k . _ -> k T.general_pattern -> _ = fun sub pat ->
         Ppat_alias (sub.pat sub pat, name)
     | Tpat_constant cst -> Ppat_constant (constant cst)
     | Tpat_tuple list ->
-        Ppat_tuple (List.map (sub.pat sub) list)
+        (* CR labeled tuples: update once Tpat_tuple has labels *)
+        Ppat_tuple (List.map (fun p -> None,(sub.pat sub p)) list)
```

Comment:
Leo always yells at me if I don't put a space after the comma in a tuple, so I guess I had better pass on the wisdom that they are mandatory.